### PR TITLE
deprecate the templating integration

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: Tests/Functional/app/ViewResponseListener/config.yml [hunk 1]
```diff
@@ -13,8 +13,6 @@ fos_rest:
         formats:
             xml:  true
             json: true
-        templating_formats:
-            html: true
     format_listener:
         rules:
             - { path: ^/, priorities: [ html, json, xml ], fallback_format: ~, prefer_extension: true }
```
Comment (yceruto) [inline Tests/Functional/app/ViewResponseListener/config.yml]: can the `html` format be removed from here too?

**Comment by lsmith77** [comment]
I think there might still be a few features lurking around in the ViewHandler that are only useful for HTML rendering. I briefly looked over the code and one feature that I noticed was `force_redirects`. This was basically about if when creating a resource if one should return the URL to the resource or redirect to the resource. The later bit I would say doesn't really make sense for a pure REST API

File: Tests/Functional/app/ViewResponseListener/config.yml (hunk 1 above)
Comment (xabbuh) [reply Tests/Functional/app/ViewResponseListener/config.yml]: I tried to do that initially but that changed the status code of the response. However, deprecating the force redirects option as suggested by @lsmith77 solve this. We now have to dedicated test cases for this feature.

**Comment by xabbuh** [comment]
@lsmith77 Good point, I also deprecated setting the `force_redirects` option.

**Pull request merged**